pkg/cloud/rgraph/rnode/tcproute/node.go

```diff
@@ -46,6 +46,11 @@ func (n *tcpRouteNode) Diff(gotNode rnode.Node) (*rnode.PlanDetails, error) {
 		return nil, fmt.Errorf("TcpRouteNode: Diff %w", err)
 	}
 
+	for i, item := range diff.Items {
+		if item.Path.String() == "*.Name" {
+			diff.Items = append(diff.Items[:i], diff.Items[i+1:]...)
```

Comment:
Name appears only once in the resources you can break the loop after you found one.

Reply:
Done.